src/styleguide/components/s-navigation-block.vue

```diff
@@ -46,27 +62,23 @@
     computed: {
       /**
        * Returns all modifiers for the component main class.
-       *
-       * @returns {Object}
        */
-      componentModifiers() {
+      componentModifiers(): IModifiers {
         return {
           hasIndent: this.hasIndent,
         };
       },
 
       /**
        * Returns an array of routes, that should be visible on the navigation.
-       *
-       * @returns {Array.<Object>}
        */
-      filteredRoutes() {
+      filteredRoutes(): object[] {
         return this.routes.filter(route => route.meta && !route.meta.hideInStyleguide);
```

Comment:
Please remove this filtering and go for `this.routes` directly. Also remove the `hideInStyleguide` field.